src/installer/src/tortuga/resourceAdapter/resourceAdapter.py

```diff
@@ -88,6 +87,8 @@ def __init__(self, addHostSession: Optional[str] = None):
 
         self._addHostSession = addHostSession
 
+        self.session = None
```

Comment:
Why can't we pass the `session` into the `__init__` method? I would prefer to see that rather than setting it after the fact, as it feels more explicit to me. Otherwise, looking at the class, it isn't apparent where the session is actually supposed to come from.

Reply:
I saw you did this to a couple other classes as well. Is there any reason why we don't want to put session into the `__init__` method on those as well?

Reply:
This was done to change the `session` argument for objects that are instantiated at app start up and reused when a request is received.